cfme/tests/cloud/test_providers.py

```diff
@@ -831,4 +1007,37 @@ def test_create_azure_vm_from_azure_image():
                   VM is completely Stopped in Azure.
         title: Create Azure VM from Azure image
     """
-    pass
+    app = appliance.IPAppliance.from_url(vm_ip)
```

Comment:
Consider using `appliance.Appliance.from_provider()` here, though you may not want to given your fixtures for `vm_ip`